src/kontrol/solc_to_k.py

```diff
@@ -776,7 +777,19 @@ def __init__(self, contract_name: str, contract_json: dict, foundry: bool = Fals
                 for rng in ref:
                     self.immutable_ranges.append((rng['start'], rng['length']))
 
+        self.link_ranges = []
+        if 'linkReferences' in deployed_bytecode:
+            for ref in deployed_bytecode['linkReferences'].values():
+                for rng_grp in ref.values():
+                    for rng in rng_grp:
+                        self.link_ranges.append((rng['start'], rng['length']))
```

Comment:
Another suggestion would be to use list comprehension instead of nested loops for better readability. Maybe something in the lines of:
```suggestion
        self.link_ranges = [
            (rng['start'], rng['length'])
            for ref in deployed_bytecode.get('linkReferences', {}).values()
            for rng_grp in ref.values()
            for rng in rng_grp
        ]
```

Reply:
Fixed.